### PR TITLE
WIP BXMSDOC-6593: Updated KIE API doc as per BAPL-1664

### PR DESCRIPTION
**Pull request by hmanwani-rh**
Added a dedicated section for Fluent API in KIE API document

See JIRA: https://issues.redhat.com/browse/BXMSDOC-6593
Impacted BAPL: https://issues.redhat.com/browse/BAPL-1664

Doc previews:
Enterprise version:
[Deploying and Managing Red Hat Process Automation Manager Services](http://file.pnq.redhat.com/~hmanwani/BXMSDOC-6593-RHPAM/#bpmn-fluent-api-con_kie-apis)

Community version
[JBPM](http://file.pnq.redhat.com/~hmanwani/BXMSDOC-6593-JBPM/#bpmn-fluent-api-con_business-processes)

### CODE REVIEW

File: doc-content/jbpm-docs/src/main/asciidoc/BPMN2/bpmn-fluent-api-con.adoc [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                       // connections
+                       .connection(1,
+                                   2)
+                       .connection(2,
+                                   3)
+                       .build();
+----
+--
++
+In this example, the static `createProcess()` method is called from the `RuleFlowProcessFactory` class. The `createProcess()` method creates a new process and returns the `RuleFlowProcessFactory` that is used to create the process.
```
Comment (fjtirado) [inline doc-content/jbpm-docs/src/main/asciidoc/BPMN2/bpmn-fluent-api-con.adoc]: Please update this section to be aligned with the community one and the text above

Comment (hmanwani-rh) [reply doc-content/jbpm-docs/src/main/asciidoc/BPMN2/bpmn-fluent-api-con.adoc]: Done

**Review by fjtirado — APPROVED** [review]
LGTM

**Comment by hmanwani-rh** [comment]
> LGTM

thanks @fjtirado

**Review by MarianMacik — APPROVED** [review]
Looks good to me.

**Pull request merged**